src/FsAutoComplete.Core/State.fs

```diff
@@ -47,7 +49,8 @@ type State =
       CancellationTokens = ConcurrentDictionary()
       NavigationDeclarations = ConcurrentDictionary()
       ScriptProjectOptions = ConcurrentDictionary()
-      ColorizationOutput = false }
+      ColorizationOutput = false
+      WorkspaceStateDirectory = workspaceStateDir }
```

Comment:
it made sense to put this on `State` for easy access, since `State` is our central location for workspace state